dll/win32/winmm/midimap/midimap.c

```diff
@@ -471,7 +471,7 @@ DWORD WINAPI MIDIMAP_modMessage(UINT wDevID, UINT wMsg, DWORD_PTR dwUser,
 	/* FIXME: Pretend this is supported */
 	return 0;
 
-    case MODM_OPEN:	 	return modOpen		((LPDWORD)dwUser,      (LPMIDIOPENDESC)dwParam1,dwParam2);
+    case MODM_OPEN: return modOpen((DWORD_PTR *)dwUser, (LPMIDIOPENDESC)dwParam1, dwParam2);
```

Comment:
Please keep the original formatting as much as possible (granted, it's not terrible, but whatever...)

Reply:
> keep the original formatting

I did not sync lines with formatting differences only,
but why not fully sync that line?

Reply:
checked, indeed they changed it.